src/Analyzer.TemplateProcessor.UnitTests/ArmTemplateProcessorTests.cs

```diff
@@ -418,13 +527,6 @@ public void CopyResourceDependants_ValidChildResourceDependsOnByNameAndResourceI
                 ""dependsOn"": [ ""someResource"" ],
                 ""properties"": { }
             }", DisplayName = "Child resource depends on resource outside template scope")]
-        [DataRow(@"{
-                ""type"": ""Microsoft.Network/networkInterfaces"",
-                ""apiVersion"": ""2018-10-01"",
-                ""name"": ""simpleLinuxVMNetInt"",
-                ""location"": ""westus"",
-                ""properties"": { }
-            }", DisplayName = "Child resource depends on not specified")]
```

Comment:
Why was this test removed?

Reply:
I realized it wasn't necessary. I had to update the test to call a function higher in the stack. Once this happened, the resource objects all had an empty dependsOn added to them after processing.

Reply:
I guess I won't block on it, but I think the test still has value - it verifies this scenario is covered in the parsing.  The implementation depends on the ARM Deployment library setting a default empty array, but that's a detail that's abstracted from the test.  i.e. the test doesn't care *how* it's accounted for, just that it is in some way.

Reply:
It seemed like unnecessary work to add the expectation of dependsOn being added when comparing the json at the end of the test. That's why I removed the test instead of updating it.

Reply:
Oh, now I see what's going on.

I get the desire to simplify the tests.  I think we should when we can.  In this case though, doesn't it seem like it's removing a test scenario in favor of simplifying the test?  Now it looks like the parsing library requires "dependsOn" to be specified.

Reply:
Added the test back and updated the test